myhpom/templates/myhpom/upload/requirements.html

```diff
@@ -13,12 +13,134 @@
 {% endblock widget_breadcrumb_items %}
```

Comment:
The additions to this file are a mix of space and tab delineated. Use space delineated (4 spaces per tab stop).

Reply:
good to know, thanks. and now I've updated my Sublime settings to make this automatic.